fmt

```diff
@@ -1,367 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I didn't manage to find out why this file was introduced. Currently it seems to work without it.